tests/testthat/test-as_polars.R

```diff
@@ -391,18 +391,12 @@ patrick::with_parameters_test_that("clock package class support",
       as.POSIXct(clock_sys_time, tz = "Asia/Kolkata")
     )
 
-    # Test on other time zone
-    withr::with_envvar(
-      new = c(TZ = "Europe/Paris"),
-      {
-        expect_equal(as.POSIXct(as.vector(pl_naive_time)), as.POSIXct(clock_naive_time))
-        expect_equal(as.POSIXct(as.vector(pl_zoned_time_1)), as.POSIXct(clock_zoned_time_1))
-        expect_equal(as.POSIXct(as.vector(pl_sys_time)), as.POSIXct(clock_sys_time, tz = "UTC"))
-        expect_equal(
-          as.POSIXct(as.vector(pl_sys_time), tz = "Asia/Kolkata"),
-          as.POSIXct(clock_sys_time, tz = "Asia/Kolkata")
-        )
-      }
+    expect_equal(as.POSIXct(as.vector(pl_naive_time)), as.POSIXct(clock_naive_time))
+    expect_equal(as.POSIXct(as.vector(pl_zoned_time_1)), as.POSIXct(clock_zoned_time_1))
```

Comment:
I am really confused. What are these tests testing?
Is it guaranteed that these tests will work with other systems' time zone settings?

Reply:
Yeah actually those could be removed since they're identical to those a few lines above. The only difference was that they were wrapped in `withr::with_envvar()`.

>  Is it guaranteed that these tests will work with other systems' time zone settings?

I don't know, but as we have seen in #1188, setting `TZ` wasn't enough to check this so I don't know the path to follow here. Do you or @ju6ge have any ideas on this?